audiophiler/templates/main.html

```diff
@@ -80,6 +80,36 @@ <h6>Owner: {{ file.author }} uploaded {{ get_date_modified(s3_bucket, file.file_
             {% if not loop.index0 is divisibleby 2 %}
                 </div>
             {% endif %}
+            {% if loop.index0 is divisibleby 2 and loop.last %}
+                </div>
+            {% endif %}
```

Comment:
```suggestion
            {% if not loop.index0 is divisibleby 2 or loop.last %}
                </div>
            {% endif %}
```
nit: this could be a bit more compact I think